pkgs/mime/doc/mime_types.md

```diff
@@ -0,0 +1,784 @@
+
+## Supported MIME types
+
+<!-- table -->
+
+| MIME type | Default ext | Additional exts |
+| - | - | - |
```

Comment:
Github Markdown tables say ["there must be at least three hyphens in each column of the header row"](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables#:~:text=There%20must%20be%20at%20least%20three%20hyphens%20in%20each%20column%20of%20the%20header%20row.).
I'm not sure that's *true*, it seems to recognize a single `-` too, but might as well follow the spec.

Consider being explicit about the alignment (I guess no alignment defaults to left-aligned, but why take chances).

```md
|:--- |:--- |:--- |
```

Reply:
Good catch! I think my other markdown tables use `---`; not sure where I cargo culted this example from.